Bluepill-cli/Bluepill-cli/Simulator/SimulatorMonitor.m

```diff
@@ -245,10 +245,10 @@ - (void)stopTestsWithErrorMessage:(NSString *)message forTestName:(NSString *)te
     if (!self.config.onlyRetryFailed) {
         [self updateExecutedTestCaseList:testName inClass:testClass];
     }
-
     if (![[self.device stateString] isEqualToString:@"Shutdown"] && !self.config.testing_NoAppWillRun) {
         [BPUtils printInfo:ERROR withString:@"Will kill the process with appPID: %d", self.appPID];
         NSAssert(self.appPID > 0, @"Failed to find a valid PID");
+        [BPUtils printInfo:ERROR withString:[BPUtils runShell:[NSString stringWithFormat:@"/usr/bin/sample %d", self.appPID]]];
```

Comment:
save to output directory instead of dumping in console

Reply:
This only happens when there is app hang, and would be very useful debug info in console when app hangs.

Reply:
I would print in the console the location of the file and pass the `-f` flag to the `sample` command to save the actual crash dump in the output directory.